AllReadyApp/Web-App/AllReady.UnitTest/Controllers/CampaignControllerTests.cs

```diff
@@ -343,6 +410,26 @@ protected override string UrlEncode(string value)
             {
                 return urlEncodedValue;
             }
+        }
+
+        private class FakeUserManager : UserManager<ApplicationUser>
+        {
+            public FakeUserManager()
+                : base(new Mock<IUserStore<ApplicationUser>>().Object,
+                      new Mock<IOptions<IdentityOptions>>().Object,
+                      new Mock<IPasswordHasher<ApplicationUser>>().Object,
+                      new IUserValidator<ApplicationUser>[0],
+                      new IPasswordValidator<ApplicationUser>[0],
+                      new Mock<ILookupNormalizer>().Object,
+                      new Mock<IdentityErrorDescriber>().Object,
+                      new Mock<IServiceProvider>().Object,
+                      new Mock<ILogger<UserManager<ApplicationUser>>>().Object)
+            { }
+
+            public override Task<ApplicationUser> GetUserAsync(ClaimsPrincipal user)
+            {
+                return Task.FromResult(new ApplicationUser { Id = "123" });
+            }
         }
```

Comment:
In this case I need `FakeUserManager` and its method `GetUserAsync` to get the user and pass its Id, I should open an issue for it.